### PR TITLE
Version 1.3

### PR DESCRIPTION
**Pull request by jdevalk**
Please review:

* Enhancements:
    * Add option to restrict comments that are too long, next to too small.
    * Preserve the active tab when saving settings.
    * Remove [...] from pingback / trackback excerpt in cleaned emails as that's already included by core.
    * Replace link to ARIN with link to ip-lookup.net for Whois on IP.
    * Refactored code for readability and code quality.

### CODE REVIEW

File: admin/views/config-page.php [hunk 1]
```diff
@@ -42,15 +42,45 @@
...
+                <p><?php _e( 'Users that try to submit a comment longer than the length you set below will get an error immediately. The text of that error is specified below too.', 'yoast-comment-hacks' ); ?></p>
+                <table class="form-table">
+                    <tr valign="top">
+                        <th scrope="row">
+                            <label
+                                    for="maxcomlength"><?php _e( 'Maximum length', 'yoast-comment-hacks' ); ?>
+                            </label>
+                        </th>
+                        <td>
+                            <input type="number" class="small-text" min="5" max="255"
```
Comment (moorscode) [inline admin/views/config-page.php]: A maximum of 255 characters might feel restricted for certain websites.
I assume this is based on SEO specifications, if so I would opt for adding a little contextual explanation about why this maximum value restriction is chosen.

Comment (moorscode) [inline admin/views/config-page.php]: I see the default is set to `1500`, so I guess the value should be higher?

**Comment by moorscode** [comment]
CR done.

File: admin/views/config-page.php (hunk 1 above)
Comment (jdevalk) [inline admin/views/config-page.php]: Agreed, removing max

**Pull request merged**